### PR TITLE
FIX: 1087 (Step backward/forward do not work)

### LINKED ISSUE #1087: Step Forward/Backwards Not Working
**Describe the bug**
Step Forward/Backwards move routes simply do not work and will hold the player in place indefinitely instead.

**Version Encountered**
0.7.0.129

**To Reproduce**
1. Make an event with a move route
2: The Move Route must be "step backwards or step forward"
3: Your character will be held like a "hold player" and won't step back/forward at all

**Expected behavior**
Step forward/backward

**Screenshots**

**Logs**
If there are error logs please attach them below.

**Operating System:**
Windows 10

**Additional context**
Similar post was made last year on this..: https://www.ascensiongamedev.com/topic/6196-step-backwardforward-command-for-player-not-working/?do=findComment&comment=52622

### PR DESCRIPTION
**Pull request by AlexVild**
Resolves #1087

### CODE REVIEW

File: Intersect.Server/Entities/Entity.cs [hunk 1]
```diff
@@ -610,9 +610,10 @@ protected virtual bool ProcessMoveRoute(Player forPlayer, long timeMs)
 
                         break;
                     case MoveRouteEnum.StepForward:
-                        if (CanMove(Dir) > -1)
+                        var x = CanMove(Dir);
```
Comment (Cheshire92) [inline Intersect.Server/Entities/Entity.cs:613]: Presuming this used to be a debug line?

Comment (AlexVild) [reply Intersect.Server/Entities/Entity.cs:613]: SIGH yep, will remove

Comment (AlexVild) [reply Intersect.Server/Entities/Entity.cs]: donezo

**Review by Arufonsu — APPROVED** [review]
Tested, this fixes the linked issue for sure 👍🏽 

https://user-images.githubusercontent.com/17498701/156942311-d8590061-eb41-45f5-88c1-eacfe44eed73.mp4

**Review by Cheshire92 — APPROVED** [review]

**Pull request merged**